README.md

```diff
@@ -90,4 +90,4 @@ We recommend to customize the user agent. If you are building a library or integ
 
 ## License
 
-Copyright (c) 2010-2020 DNSimple Corporation. This is Free Software distributed under the MIT license.
```

Comment:
Beware this is not the only place to change.

```
➜  dnsimple-ruby git:(main) ag '2010-2020'
README.md
93:Copyright (c) 2010-2020 DNSimple Corporation. This is Free Software distributed under the MIT license.

LICENSE.txt
3:Copyright (c) 2010-2020 DNSimple Corporation
```

Reply:
Addressed at 44c4166. Thanks!